### PR TITLE
Pn 10135

### PR DESCRIPTION
**Pull request by sahdevsaliya**

### CODE REVIEW

File: PAS_Database/dbo/Views/vw_MasterDiscountType.sql [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
       ,MD.[MasterCompanyId]
       ,MD.[CreatedBy]
       ,MD.[CreatedDate]
       ,MD.[UpdatedBy]
       ,MD.[UpdatedDate]
       ,MD.[IsActive]
       ,MD.[IsDeleted]
-  FROM [dbo].[MasterDiscountType] MD LEFT JOIN [dbo].[GLAccount] GL ON MD.GLAccountId=GL.GLAccountId
\ No newline at end of file
+  FROM [dbo].[MasterDiscountType] MD
```
Comment (projectpas) [inline PAS_Database/dbo/Views/vw_MasterDiscountType.sql]: Add NOLOCK

**Comment by amitghediyasilverxis** [comment]
Reviewed

**Comment by projectpas** [comment]
@amitghediyasilverxis , where is the no lock did you verifyed ?

File: PAS_Database/dbo/Views/vw_MasterDiscountType.sql [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
       ,MD.[CreatedBy]
       ,MD.[CreatedDate]
       ,MD.[UpdatedBy]
       ,MD.[UpdatedDate]
       ,MD.[IsActive]
       ,MD.[IsDeleted]
-  FROM [dbo].[MasterDiscountType] MD LEFT JOIN [dbo].[GLAccount] GL ON MD.GLAccountId=GL.GLAccountId
\ No newline at end of file
+  FROM [dbo].[MasterDiscountType] MD 
+  LEFT JOIN [dbo].[GLAccount] GL ON MD.GLAccountId=GL.GLAccountId
```
Comment (amitghediyasilverxis) [inline PAS_Database/dbo/Views/vw_MasterDiscountType.sql]: Add nolock

File: PAS_Database/dbo/Views/vw_MasterDiscountType.sql (hunk 1 above)
Comment (amitghediyasilverxis) [inline PAS_Database/dbo/Views/vw_MasterDiscountType.sql]: Add nolock

**Review by projectpas — APPROVED** [review]

**Pull request merged**